pkg/cvo/sync_worker.go

```diff
@@ -494,6 +502,7 @@ func (w *SyncWorker) syncOnce(ctx context.Context, work *SyncWork, maxWorkers in
 			return err
 		}
 
+		w.eventRecorder.Eventf(cvoObjectRef, corev1.EventTypeNormal, "VerifyPayload", "verifying payload version=%q image=%q", update.Version, update.Image)
```

Comment:
I kind of expected to see the "unable to verify payload event"

Reply:
> I kind of expected to see the "unable to verify payload event" in case of error.

added events for error cases